### ISSUE TITLE
Configure HTTP Send on FormAssemby form for AI Intersections Database newsletter sign-ups

### ISSUE THREAD

**Issue by data-sync-user** [MozillaFoundation/foundation.mozilla.org#12206]
Request created by Kenrya Rankin at [https://mozilla-hub.atlassian.net/jira/servicedesk/projects/MOFOTP/issues/MOFOTP-178](https://mozilla-hub.atlassian.net/jira/servicedesk/projects/MOFOTP/issues/MOFOTP-178).

### Request description
Configuration of AI intersection database form to connect with the Basket API. Draft from (cloned from Petition form) is available here: [https://mozillafoundation.tfaforms.net/forms/builder/5.0.0/53](https://mozillafoundation.tfaforms.net/forms/builder/5.0.0/53)

I did not delete fields on the form yet as I was not sure which are needed/not needed for the HTTP connector and embedding on the website. Currently, the insights newsletter form only collects email, country, language and has a checkbox for agreeing to the privacy policy ( as seen here: [https://foundation.mozilla.org/en/insights/](https://foundation.mozilla.org/en/insights/|smart-link)

)

### Timelines: Ideally, when would you like this work complete? If there is a soft launch date required for press or partners, please use that.
NA

### What are the metrics and/or Key Performance Indicators you’re hoping to impact with this request?
NA

### What is the audience you are trying to reach with this request?
NA

### Do you have any assets that may help bring us up to speed? (Please upload any Images, briefs, documentation inspiration, screenshots of bugs, etc.)
NA

- - -
Thank you for your submission! We will review your submission and a member of the Technology Platforms team will be in touch with you soon.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/TP1-501)

**Comment by data-sync-user** [comment]
➤ Daniel Miranda commented:

Hi Everyone! The form at https://mozillafoundation.tfaforms.net/forms/builder/5.0.0/53 ( https://mozillafoundation.tfaforms.net/forms/builder/5.0.0/53 ) is now set up and ready to be embedded 👍 

Here is the embed code generated by FormAssembly:

Paste this in the <head> section


    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="referrer" content="no-referrer-when-downgrade">
    
    <script type="text/javascript" data-for="FA__DOMContentLoadedEventDispatch" src="https://mozillafoundation.tfaforms.net/js/FA__DOMContentLoadedEventDispatcher.js" defer></script>
            <script type="text/javascript">
        document.addEventListener("FA__DOMContentLoaded", function(){
            const FORM_TIME_START = Math.floor((new Date).getTime()/1000);
            let formElement = document.getElementById("tfa_0");
            if (null === formElement) {
                formElement = document.getElementById("0");
            }
            let appendJsTimerElement = function(){
                let formTimeDiff = Math.floor((new Date).getTime()/1000) - FORM_TIME_START;
                let cumulatedTimeElement = document.getElementById("tfa_dbCumulatedTime");
                if (null !== cumulatedTimeElement) {
                    let cumulatedTime = parseInt(cumulatedTimeElement.value);
                    if (null !== cumulatedTime && cumulatedTime > 0) {
                        formTimeDiff += cumulatedTime;
                    }
                }
                let jsTimeInput = document.createElement("input");
                jsTimeInput.setAttribute("type", "hidden");
                jsTimeInput.setAttribute("value", formTimeDiff.toString());
                jsTimeInput.setAttribute("name", "tfa_dbElapsedJsTime");
                jsTimeInput.setAttribute("id", "tfa_dbElapsedJsTime");
                jsTimeInput.setAttribute("autocomplete", "off");
                if (null !== formElement) {
                    formElement.appendChild(jsTimeInput);
                }
            };
            if (null !== formElement) {
                if(formElement.addEventListener){
                    formElement.addEventListener('submit', appendJsTimerElement, false);
                } else if(formElement.attachEvent){
                    formElement.attachEvent('onsubmit', appendJsTimerElement);
                }
            }
        });
    </script>

    <link href="https://mozillafoundation.tfaforms.net/dist/form-builder/5.0.0/wforms-layout.css?v=8b5c9ffd0b6fc88dc72529725004f6b69b9e0ae8" rel="stylesheet" type="text/css" />

    <link href="https://mozillafoundation.tfaforms.net/uploads/themes/theme-23.css" rel="stylesheet" type="text/css" />
    <link href="https://mozillafoundation.tfaforms.net/dist/form-builder/5.0.0/wforms-jsonly.css?v=8b5c9ffd0b6fc88dc72529725004f6b69b9e0ae8" rel="alternate stylesheet" title="This stylesheet activated by javascript" type="text/css" />
    <script type="text/javascript" src="https://mozillafoundation.tfaforms.net/wForms/3.11/js/wforms.js?v=8b5c9ffd0b6fc88dc72529725004f6b69b9e0ae8"></script>
    <script type="text/javascript">
        wFORMS.behaviors.prefill.skip = false;
    </script>
        <script type="text/javascript" src="https://mozillafoundation.tfaforms.net/wForms/3.11/js/localization-en_US.js?v=8b5c9ffd0b6fc88dc72529725004f6b69b9e0ae8"></script>
And this goes where you want the form to render:


<div class="wFormContainer" style="max-width: 100%; width:auto;">
    <div class="wFormHeader"></div>
    <style type="text/css"></style><div class=""><div class="wForm" id="53-WRPR" data-language="en_US" dir="ltr">
<div class="codesection" id="code-53"></div>
<form method="post" action="https://mozillafoundation.tfaforms.net/api_v2/workflow/processor" class=" labelsAbove" id="53" role="form">
<input type="hidden" id="tfa_500" name="tfa_500" value="" class=""><input type="hidden" id="tfa_498" name="tfa_498" value="" class=""><input type="hidden" id="tfa_510" name="tfa_510" value="" class="formula=LANGCODE"><input type="hidden" id="tfa_501" name="tfa_501" value="internet-health-report-group" class=""><div class="oneField field-container-D    " id="tfa_28-D">
<label id="tfa_28-L" class="label preField reqMark" for="tfa_28">First Name</label><br><div class="inputWrapper"><input aria-required="true" type="text" id="tfa_28" name="tfa_28" value="" title="First Name" class="required"></div>
</div>
<div class="oneField field-container-D    " id="tfa_30-D">
<label id="tfa_30-L" class="label preField reqMark" for="tfa_30">Last Name</label><br><div class="inputWrapper"><input aria-required="true" type="text" id="tfa_30" name="tfa_30" value="" title="Last Name" class="required"></div>
</div>
<div class="oneField field-container-D    " id="tfa_31-D">
<label id="tfa_31-L" class="label preField reqMark" for="tfa_31">Email</label><br><div class="inputWrapper"><input aria-required="true" type="text" id="tfa_31" name="tfa_31" value="" title="Email" class="validate-email required"></div>
</div>
<div class="oneField field-container-D    " id="tfa_254-D">
<label id="tfa_254-L" class="label preField reqMark" for="tfa_254">Country</label><br><div class="inputWrapper"><select aria-required="true" id="tfa_254" name="tfa_254" title="Country" class="required"><option value="">Please select...</option>
<option value="tfa_255" id="tfa_255" class="">Afghanistan</option>
<option value="tfa_256" id="tfa_256" class="">Albania</option>
<option value="tfa_257" id="tfa_257" class="">Algeria</option>
<option value="tfa_258" id="tfa_258" class="">American Samoa</option>
<option value="tfa_259" id="tfa_259" class="">Andorra</option>
<option value="tfa_260" id="tfa_260" class="">Angola</option>
<option value="tfa_261" id="tfa_261" class="">Anguilla</option>
<option value="tfa_262" id="tfa_262" class="">Antarctica</option>
<option value="tfa_263" id="tfa_263" class="">Antigua and Barbuda</option>
<option value="tfa_264" id="tfa_264" class="">Argentina</option>
<option value="tfa_265" id="tfa_265" class="">Armenia</option>
<option value="tfa_266" id="tfa_266" class="">Aruba</option>
<option value="tfa_267" id="tfa_267" class="">Australia</option>
<option value="tfa_268" id="tfa_268" class="">Austria</option>
<option value="tfa_269" id="tfa_269" class="">Azerbaijan</option>
<option value="tfa_270" id="tfa_270" class="">Bahamas</option>
<option value="tfa_271" id="tfa_271" class="">Bahrain</option>
<option value="tfa_272" id="tfa_272" class="">Bangladesh</option>
<option value="tfa_273" id="tfa_273" class="">Barbados</option>
<option value="tfa_274" id="tfa_274" class="">Belarus</option>
<option value="tfa_275" id="tfa_275" class="">Belgium</option>
<option value="tfa_276" id="tfa_276" class="">Belize</option>
<option value="tfa_277" id="tfa_277" class="">Benin</option>
<option value="tfa_278" id="tfa_278" class="">Bermuda</option>
<option value="tfa_279" id="tfa_279" class="">Bhutan</option>
<option value="tfa_280" id="tfa_280" class="">Bolivia</option>
<option value="tfa_281" id="tfa_281" class="">Bosnia and Herzegovina</option>
<option value="tfa_282" id="tfa_282" class="">Botswana</option>
<option value="tfa_283" id="tfa_283" class="">Bouvet Island</option>
<option value="tfa_284" id="tfa_284" class="">Brazil</option>
<option value="tfa_285" id="tfa_285" class="">British Indian Ocean Territory</option>
<option value="tfa_286" id="tfa_286" class="">Brunei</option>
<option value="tfa_287" id="tfa_287" class="">Bulgaria</option>
<option value="tfa_288" id="tfa_288" class="">Burkina Faso</option>
<option value="tfa_289" id="tfa_289" class="">Burundi</option>
<option value="tfa_290" id="tfa_290" class="">Cambodia</option>
<option value="tfa_291" id="tfa_291" class="">Cameroon</option>
<option value="tfa_292" id="tfa_292" class="">Canada</option>
<option value="tfa_293" id="tfa_293" class="">Cape Verde</option>
<option value="tfa_294" id="tfa_294" class="">Cayman Islands</option>
<option value="tfa_295" id="tfa_295" class="">Central African Republic</option>
<option value="tfa_296" id="tfa_296" class="">Chad</option>
<option value="tfa_297" id="tfa_297" class="">Chile</option>
<option value="tfa_298" id="tfa_298" class="">China</option>
<option value="tfa_299" id="tfa_299" class="">Christmas Island</option>
<option value="tfa_300" id="tfa_300" class="">Cocos ( Keeling ) Islands</option>
<option value="tfa_301" id="tfa_301" class="">Colombia</option>
<option value="tfa_302" id="tfa_302" class="">Comoros</option>
<option value="tfa_303" id="tfa_303" class="">Congo</option>
<option value="tfa_304" id="tfa_304" class="">Cook Islands</option>
<option value="tfa_305" id="tfa_305" class="">Costa Rica</option>
<option value="tfa_306" id="tfa_306" class="">Côte d ' Ivoire</option>
<option value="tfa_307" id="tfa_307" class="">Croatia ( Hrvatska )</option>
<option value="tfa_308" id="tfa_308" class="">Cuba</option>
<option value="tfa_309" id="tfa_309" class="">Cyprus</option>
<option value="tfa_310" id="tfa_310" class="">Czech Republic</option>
<option value="tfa_311" id="tfa_311" class="">Congo ( DRC )</option>
<option value="tfa_312" id="tfa_312" class="">Denmark</option>
<option value="tfa_313" id="tfa_313" class="">Djibouti</option>
<option value="tfa_314" id="tfa_314" class="">Dominica</option>
<option value="tfa_315" id="tfa_315" class="">Dominican Republic</option>
<option value="tfa_316" id="tfa_316" class="">East Timor</option>
<option value="tfa_317" id="tfa_317" class="">Ecuador</option>
<option value="tfa_318" id="tfa_318" class="">Egypt</option>
<option value="tfa_319" id="tfa_319" class="">El Salvador</option>
<option value="tfa_320" id="tfa_320" class="">Equatorial Guinea</option>
<option value="tfa_321" id="tfa_321" class="">Eritrea</option>
<option value="tfa_322" id="tfa_322" class="">Estonia</option>
<option value="tfa_323" id="tfa_323" class="">Ethiopia</option>
<option value="tfa_324" id="tfa_324" class="">Falkland Islands ( Islas Malvinas )</option>
<option value="tfa_325" id="tfa_325" class="">Faroe Islands</option>
<option value="tfa_326" id="tfa_326" class="">Fiji Islands</option>
<option value="tfa_327" id="tfa_327" class="">Finland</option>
<option value="tfa_328" id="tfa_328" class="">France</option>
<option value="tfa_329" id="tfa_329" class="">French Guiana</option>
<option value="tfa_330" id="tfa_330" class="">French Polynesia</option>
<option value="tfa_331" id="tfa_331" class="">French Southern and Antarctic Lands</option>
<option value="tfa_332" id="tfa_332" class="">Gabon</option>
<option value="tfa_333" id="tfa_333" class="">Gambia</option>
<option value="tfa_334" id="tfa_334" class="">Georgia</option>
<option value="tfa_335" id="tfa_335" class="">Germany</option>
<option value="tfa_336" id="tfa_336" class="">Ghana</option>
<option value="tfa_337" id="tfa_337" class="">Gibraltar</option>
<option value="tfa_338" id="tfa_338" class="">Greece</option>
<option value="tfa_339" id="tfa_339" class="">Greenland</option>
<option value="tfa_340" id="tfa_340" class="">Grenada</option>
<option value="tfa_341" id="tfa_341" class="">Guadeloupe</option>
<option value="tfa_342" id="tfa_342" class="">Guam</option>
<option value="tfa_343" id="tfa_343" class="">Guatemala</option>
<option value="tfa_344" id="tfa_344" class="">Guinea</option>
<option value="tfa_345" id="tfa_345" class="">Guinea-Bissau</option>
<option value="tfa_346" id="tfa_346" class="">Guyana</option>
<option value="tfa_347" id="tfa_347" class="">Haiti</option>
<option value="tfa_348" id="tfa_348" class="">Heard Island and McDonald Islands</option>
<option value="tfa_349" id="tfa_349" class="">Honduras</option>
<option value="tfa_350" id="tfa_350" class="">Hong Kong SAR</option>
<option value="tfa_351" id="tfa_351" class="">Hungary</option>
<option value="tfa_352" id="tfa_352" class="">Iceland</option>
<option value="tfa_353" id="tfa_353" class="">India</option>
<option value="tfa_354" id="tfa_354" class="">Indonesia</option>
<option value="tfa_355" id="tfa_355" class="">Iran</option>
<option value="tfa_356" id="tfa_356" class="">Iraq</option>
<option value="tfa_357" id="tfa_357" class="">Ireland</option>
<option value="tfa_358" id="tfa_358" class="">Israel</option>
<option value="tfa_359" id="tfa_359" class="">Italy</option>
<option value="tfa_360" id="tfa_360" class="">Jamaica</option>
<option value="tfa_361" id="tfa_361" class="">Japan</option>
<option value="tfa_362" id="tfa_362" class="">Jordan</option>
<option value="tfa_363" id="tfa_363" class="">Kazakhstan</option>
<option value="tfa_364" id="tfa_364" class="">Kenya</option>
<option value="tfa_365" id="tfa_365" class="">Kiribati</option>
<option value="tfa_366" id="tfa_366" class="">Korea</option>
<option value="tfa_367" id="tfa_367" class="">Kuwait</option>
<option value="tfa_368" id="tfa_368" class="">Kyrgyzstan</option>
<option value="tfa_369" id="tfa_369" class="">Laos</option>
<option value="tfa_370" id="tfa_370" class="">Latvia</option>
<option value="tfa_371" id="tfa_371" class="">Lebanon</option>
<option value="tfa_372" id="tfa_372" class="">Lesotho</option>
<option value="tfa_373" id="tfa_373" class="">Liberia</option>
<option value="tfa_374" id="tfa_374" class="">Libya</option>
<option value="tfa_375" id="tfa_375" class="">Liechtenstein</option>
<option value="tfa_376" id="tfa_376" class="">Lithuania</option>
<option value="tfa_377" id="tfa_377" class="">Luxembourg</option>
<option value="tfa_378" id="tfa_378" class="">Macao SAR</option>
<option value="tfa_379" id="tfa_379" class="">Macedonia, Former Yugoslav Republic of</option>
<option value="tfa_380" id="tfa_380" class="">Madagascar</option>
<option value="tfa_381" id="tfa_381" class="">Malawi</option>
<option value="tfa_382" id="tfa_382" class="">Malaysia</option>
<option value="tfa_383" id="tfa_383" class="">Maldives</option>
<option value="tfa_384" id="tfa_384" class="">Mali</option>
<option value="tfa_385" id="tfa_385" class="">Malta</option>
<option value="tfa_386" id="tfa_386" class="">Marshall Islands</option>
<option value="tfa_387" id="tfa_387" class="">Martinique</option>
<option value="tfa_388" id="tfa_388" class="">Mauritania</option>
<option value="tfa_389" id="tfa_389" class="">Mauritius</option>
<option value="tfa_390" id="tfa_390" class="">Mayotte</option>
<option value="tfa_391" id="tfa_391" class="">Mexico</option>
<option value="tfa_392" id="tfa_392" class="">Micronesia</option>
<option value="tfa_393" id="tfa_393" class="">Moldova</option>
<option value="tfa_394" id="tfa_394" class="">Monaco</option>
<option value="tfa_395" id="tfa_395" class="">Mongolia</option>
<option value="tfa_396" id="tfa_396" class="">Montserrat</option>
<option value="tfa_397" id="tfa_397" class="">Morocco</option>
<option value="tfa_398" id="tfa_398" class="">Mozambique</option>
<option value="tfa_399" id="tfa_399" class="">Myanmar</option>
<option value="tfa_400" id="tfa_400" class="">Namibia</option>
<option value="tfa_401" id="tfa_401" class="">Nauru</option>
<option value="tfa_402" id="tfa_402" class="">Nepal</option>
<option value="tfa_403" id="tfa_403" class="">Netherlands</option>
<option value="tfa_404" id="tfa_404" class="">Netherlands Antilles</option>
<option value="tfa_405" id="tfa_405" class="">New Caledonia</option>
<option value="tfa_406" id="tfa_406" class="">New Zealand</option>
<option value="tfa_407" id="tfa_407" class="">Nicaragua</option>
<option value="tfa_408" id="tfa_408" class="">Niger</option>
<option value="tfa_409" id="tfa_409" class="">Nigeria</option>
<option value="tfa_410" id="tfa_410" class="">Niue</option>
<option value="tfa_411" id="tfa_411" class="">Norfolk Island</option>
<option value="tfa_412" id="tfa_412" class="">North Korea</option>
<option value="tfa_413" id="tfa_413" class="">Northern Mariana Islands</option>
<option value="tfa_414" id="tfa_414" class="">Norway</option>
<option value="tfa_415" id="tfa_415" class="">Oman</option>
<option value="tfa_416" id="tfa_416" class="">Pakistan</option>
<option value="tfa_417" id="tfa_417" class="">Palau</option>
<option value="tfa_418" id="tfa_418" class="">Panama</option>
<option value="tfa_419" id="tfa_419" class="">Papua New Guinea</option>
<option value="tfa_420" id="tfa_420" class="">Paraguay</option>
<option value="tfa_421" id="tfa_421" class="">Peru</option>
<option value="tfa_422" id="tfa_422" class="">Philippines</option>
<option value="tfa_423" id="tfa_423" class="">Pitcairn Islands</option>
<option value="tfa_424" id="tfa_424" class="">Poland</option>
<option value="tfa_425" id="tfa_425" class="">Portugal</option>
<option value="tfa_426" id="tfa_426" class="">Puerto Rico</option>
<option value="tfa_427" id="tfa_427" class="">Qatar</option>
<option value="tfa_428" id="tfa_428" class="">Reunion</option>
<option value="tfa_429" id="tfa_429" class="">Romania</option>
<option value="tfa_430" id="tfa_430" class="">Russia</option>
<option value="tfa_431" id="tfa_431" class="">Rwanda</option>
<option value="tfa_432" id="tfa_432" class="">Samoa</option>
<option value="tfa_433" id="tfa_433" class="">San Marino</option>
<option value="tfa_434" id="tfa_434" class="">São Tomé and Prìncipe</option>
<option value="tfa_435" id="tfa_435" class="">Saudi Arabia</option>
<option value="tfa_436" id="tfa_436" class="">Senegal</option>
<option value="tfa_437" id="tfa_437" class="">Serbia and Montenegro</option>
<option value="tfa_438" id="tfa_438" class="">Seychelles</option>
<option value="tfa_439" id="tfa_439" class="">Sierra Leone</option>
<option value="tfa_440" id="tfa_440" class="">Singapore</option>
<option value="tfa_441" id="tfa_441" class="">Slovakia</option>
<option value="tfa_442" id="tfa_442" class="">Slovenia</option>
<option value="tfa_443" id="tfa_443" class="">Solomon Islands</option>
<option value="tfa_444" id="tfa_444" class="">Somalia</option>
<option value="tfa_445" id="tfa_445" class="">South Africa</option>
<option value="tfa_446" id="tfa_446" class="">South Georgia and the South Sandwich Islands</option>
<option value="tfa_447" id="tfa_447" class="">Spain</option>
<option value="tfa_448" id="tfa_448" class="">Sri Lanka</option>
<option value="tfa_449" id="tfa_449" class="">St. Helena</option>
<option value="tfa_450" id="tfa_450" class="">St. Kitts and Nevis</option>
<option value="tfa_451" id="tfa_451" class="">St. Lucia</option>
<option value="tfa_452" id="tfa_452" class="">St. Pierre and Miquelon</option>
<option value="tfa_453" id="tfa_453" class="">St. Vincent and the Grenadines</option>
<option value="tfa_454" id="tfa_454" class="">Sudan</option>
<option value="tfa_455" id="tfa_455" class="">Suriname</option>
<option value="tfa_456" id="tfa_456" class="">Svalbard and Jan Mayen</option>
<option value="tfa_457" id="tfa_457" class="">Swaziland</option>
<option value="tfa_458" id="tfa_458" class="">Sweden</option>
<option value="tfa_459" id="tfa_459" class="">Switzerland</option>
<option value="tfa_460" id="tfa_460" class="">Syria</option>
<option value="tfa_461" id="tfa_461" class="">Taiwan</option>
<option value="tfa_462" id="tfa_462" class="">Tajikistan</option>
<option value="tfa_463" id="tfa_463" class="">Tanzania</option>
<option value="tfa_464" id="tfa_464" class="">Thailand</option>
<option value="tfa_465" id="tfa_465" class="">Togo</option>
<option value="tfa_466" id="tfa_466" class="">Tokelau</option>
<option value="tfa_467" id="tfa_467" class="">Tonga</option>
<option value="tfa_468" id="tfa_468" class="">Trinidad and Tobago</option>
<option value="tfa_469" id="tfa_469" class="">Tunisia</option>
<option value="tfa_470" id="tfa_470" class="">Turkey</option>
<option value="tfa_471" id="tfa_471" class="">Turkmenistan</option>
<option value="tfa_472" id="tfa_472" class="">Turks and Caicos Islands</option>
<option value="tfa_473" id="tfa_473" class="">Tuvalu</option>
<option value="tfa_474" id="tfa_474" class="">Uganda</option>
<option value="tfa_475" id="tfa_475" class="">Ukraine</option>
<option value="tfa_476" id="tfa_476" class="">United Arab Emirates</option>
<option value="tfa_477" id="tfa_477" class="">United Kingdom</option>
<option value="tfa_478" id="tfa_478" class="">United States</option>
<option value="tfa_479" id="tfa_479" class="">United States Minor Outlying Islands</option>
<option value="tfa_480" id="tfa_480" class="">Uruguay</option>
<option value="tfa_481" id="tfa_481" class="">Uzbekistan</option>
<option value="tfa_482" id="tfa_482" class="">Vanuatu</option>
<option value="tfa_483" id="tfa_483" class="">Vatican City</option>
<option value="tfa_484" id="tfa_484" class="">Venezuela</option>
<option value="tfa_485" id="tfa_485" class="">Viet Nam</option>
<option value="tfa_486" id="tfa_486" class="">Virgin Islands ( British )</option>
<option value="tfa_487" id="tfa_487" class="">Virgin Islands</option>
<option value="tfa_488" id="tfa_488" class="">Wallis and Futuna</option>
<option value="tfa_489" id="tfa_489" class="">Yemen</option>
<option value="tfa_490" id="tfa_490" class="">Zambia</option>
<option value="tfa_491" id="tfa_491" class="">Zimbabwe</option></select></div>
</div>
<div class="oneField field-container-D    " id="tfa_499-D">
<label id="tfa_499-L" class="label preField reqMark" for="tfa_499">Language</label><br><div class="inputWrapper"><select aria-required="true" id="tfa_499" name="tfa_499" title="Language" class="calc-LANGCODE required"><option value="">Please select...</option>
<option value="tfa_511" id="tfa_511" class="calcval-en">English</option>
<option value="tfa_512" id="tfa_512" class="calcval-es">Español</option>
<option value="tfa_513" id="tfa_513" class="calcval-de">Deutsch</option>
<option value="tfa_514" id="tfa_514" class="calcval-fr">Français</option>
<option value="tfa_515" id="tfa_515" class="calcval-pl">Polski</option>
<option value="tfa_516" id="tfa_516" class="calcval-pt-BR">Português</option></select></div>
</div>
<div class="oneField field-container-D  labelsAbove  " id="tfa_492-D" role="group" aria-labelledby="tfa_492-L" data-tfa-labelledby="-L tfa_492-L">
<label id="tfa_492-L" class="label preField reqMark" data-tfa-check-label-for="tfa_492" aria-label="Privacy Agreement   required">Privacy Agreement</label><br><div class="inputWrapper"><span id="tfa_492" class="choices vertical required"><span class="oneChoice"><input type="checkbox" value="tfa_493" class="calc-DefaultName" id="tfa_493" name="tfa_493" aria-labelledby="tfa_493-L" data-tfa-labelledby="tfa_492-L tfa_493-L" data-tfa-parent-id="tfa_492"><label class="label postField" id="tfa_493-L" for="tfa_493"><span class="input-checkbox-faux"></span>I'm okay with Mozilla handling my info as explained in this <a target="_blank" href="https://www.mozilla.org/privacy/websites/">Privacy Notice</a>.</label></span></span></div>
</div>
<div class="oneField field-container-D    " id="tfa_494-D" role="group" aria-labelledby="tfa_494-L" data-tfa-labelledby="-L tfa_494-L">
<label id="tfa_494-L" class="label preField " data-tfa-check-label-for="tfa_494">Email Subscription</label><br><div class="inputWrapper"><span id="tfa_494" class="choices vertical "><span class="oneChoice"><input type="checkbox" value="tfa_495" class="" id="tfa_495" name="tfa_495" aria-labelledby="tfa_495-L" data-tfa-labelledby="tfa_494-L tfa_495-L" data-tfa-parent-id="tfa_494"><label class="label postField" id="tfa_495-L" for="tfa_495"><span class="input-checkbox-faux"></span>Yes, I want to receive email updates about Mozilla’s campaigns.</label></span></span></div>
</div>
<div class="actions" id="53-A" data-contentid="submit_button"><input type="submit" data-label="Add my name" class="primaryAction" id="submit_button" value="Add my name"></div>
<div style="clear:both"></div>
<input type="hidden" value="53" name="tfa_dbFormId" id="tfa_dbFormId"><input type="hidden" value="" name="tfa_dbResponseId" id="tfa_dbResponseId"><input type="hidden" value="d42e71b446cca368d0a67d1b0640d7da" name="tfa_dbControl" id="tfa_dbControl"><input type="hidden" value="" name="tfa_dbWorkflowSessionUuid" id="tfa_dbWorkflowSessionUuid"><input type="hidden" value="" name="tfa_noOverWriteFields" id="tfa_noOverWriteFields"><input type="hidden" value="20" name="tfa_dbVersionId" id="tfa_dbVersionId"><input type="hidden" value="" name="tfa_switchedoff" id="tfa_switchedoff">
</form>
</div></div><div class="wFormFooter"><p class="supportInfo"><br></p></div>
  <p class="supportInfo" >
      </p>
 </div>

<script
    id="open-telemetry-script"
    type="text/javascript"
    src="https://mozillafoundation.tfaforms.net/dist/open-telemetry/open-telemetry.3e6c1bedaa7fb4452dd0.js"
    data-customer-id="170610"
    data-exporter-url="https://us-east-1-otel.formassembly.com/v1/traces"
    data-exporter-console="0"
></script>
<script src="https://mozillafoundation.tfaforms.net/api_v2/sst/copy-and-paste"></script>

**Comment by data-sync-user** [comment]
➤ Daniel Miranda commented:

Alternatively, it looks like FormAssembly provides some javascript that will also load the form in, if styling is not an issue: 

<script src="https://mozillafoundation.tfaforms.net/publish/53" data-qp-target-id="" defer></script>

**Comment by data-sync-user** [comment]
➤ Daniel Miranda commented:

One last thing that I want to call out are two fields that can be utilized to make the user experience better: thank you URL and source URL.

Thank you URL:

At the moment, a successful submission of the form assembly form will redirect users to an unstyled page saying “thank you”. If stakeholders can create a styled page on their site that has a thank you message, we can instead redirect users here to avoid users being redirected to an unstyled, formassembly hosted page. This would need to be set up on the FormAssembly side. If stakeholders end up wanting to go this route, I would be happy to set up the redirect once a thank you page is in place.

Source URL:

This field can be set to the URL of the site that the form will be hosted on, that way stakeholders can track how many subscriptions are coming from this site.

This would have to be set up on FormAssembly also, but if stakeholders provide a URL, I would be happy to set this up 👍

**Comment by data-sync-user** [comment]
➤ Tyler Denton commented:

Handed off to Social Driver, waiting on response

**Comment by data-sync-user** [comment]
➤ Tyler Denton commented:

Social Driver took a whack at this while I was out and missed some steps in their process that would resolve this ticket. This ticket is now linked to the prerequisite ticket.
